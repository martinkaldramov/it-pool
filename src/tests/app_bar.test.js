import React from 'react';
import { shallow } from 'enzyme';
import ApplicationBar from '../containers/app_bar';

describe('ApplicationBar tests', () => {
  const wrap = shallow(<ApplicationBar />);

  it('should render successfully', () => {
    expect(wrap).toMatchSnapshot();
  });
});
