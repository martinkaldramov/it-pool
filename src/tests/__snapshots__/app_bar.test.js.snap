// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationBar tests should render successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ApplicationBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "iconElementRight": <Unknown>
        <FlatButton
                className="sign-in"
                disabled={false}
                fullWidth={false}
                label="Log In"
                labelPosition="after"
                labelStyle={Object {}}
                onKeyboardFocus={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                onTouchStart={[Function]}
                primary={false}
                secondary={false}
        />
        <RaisedButton
                className="sign-up"
                disabled={false}
                fullWidth={false}
                label="Sign Up"
                labelPosition="after"
                primary={false}
                secondary={false}
        />
        <RaisedButton
                className="post-a-job"
                disabled={false}
                fullWidth={false}
                label="Post a Job"
                labelPosition="after"
                primary={false}
                secondary={false}
        />
</Unknown>,
      "showMenuIconButton": false,
      "title": <span
        style={
                Object {
                        "cursor": "pointer",
                      }
        }
>
        IT Pool
</span>,
      "zDepth": 1,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "iconElementRight": <Unknown>
          <FlatButton
                    className="sign-in"
                    disabled={false}
                    fullWidth={false}
                    label="Log In"
                    labelPosition="after"
                    labelStyle={Object {}}
                    onKeyboardFocus={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    onTouchStart={[Function]}
                    primary={false}
                    secondary={false}
          />
          <RaisedButton
                    className="sign-up"
                    disabled={false}
                    fullWidth={false}
                    label="Sign Up"
                    labelPosition="after"
                    primary={false}
                    secondary={false}
          />
          <RaisedButton
                    className="post-a-job"
                    disabled={false}
                    fullWidth={false}
                    label="Post a Job"
                    labelPosition="after"
                    primary={false}
                    secondary={false}
          />
</Unknown>,
        "showMenuIconButton": false,
        "title": <span
          style={
                    Object {
                              "cursor": "pointer",
                            }
          }
>
          IT Pool
</span>,
        "zDepth": 1,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
